### ISSUE TITLE
Fail to load swagger-ui assets

### ISSUE THREAD

**Issue by benoittgt** [rswag/rswag#30]
Hello @domaindrivendev 

First thank you soooo much for the rewrite. I've edited my old request specs and it's work perfectly with query params. The json generated doesn't contain any errors when testing with http://editor.swagger.io/#/.

I have only one issue with swagger-ui.

When trying to go to `/api-docs` it gets lot's of missing assets like: 

```
ActionController::RoutingError (No route matches [GET] "/assets/swagger-ui/images/favicon-32x32.png")
```

Page looks like : 
<img width="233" alt="capture d ecran 2016-10-12 a 11 25 38" src="https://cloud.githubusercontent.com/assets/8417720/19304860/a7788bc2-906e-11e6-822d-e5ced61fe513.png">

also

``` sh
$ rails g rswag:ui:custom_ui                                                                             [11:26:20]
Running via Spring preloader in process 19510
Could not find generator 'rswag:ui:custom_ui'. Maybe you meant 'rswag:ui:custom', 'rswag:ui:install' or 'rswag:install'
Run `rails generate --help` for more options.
```

**Comment by domaindrivendev** [comment]
Thanks for the quick feedback. Odd that the assets aren't working. Is this dev or prod? Do u def have asset pipeline enabled - can u access other assets in your app? Which version of Rails?

Re the generator, my bad I changed to "rswag:ui:custom" but have the old name in the docs. Will correct docs

**Comment by benoittgt** [comment]
It's in dev env. Rails `4.2.5.2`
In dev env : 

```
  config.assets.compress = false
  config.assets.digest = false
```

No problem with the other assets of the app.

**Comment by domaindrivendev** [comment]
Tried with Rails 4.2.0 and still can't repro. Do you have the following line in your routes.rb:

```
mount Rswag::Ui::Engine, at: 'api-docs'
```

Although I don't really think that should matter

**Comment by domaindrivendev** [comment]
@benoittgt - can you try upgrading to 1.0.1 and let me know if it works?

**Comment by benoittgt** [comment]
@domaindrivendev it's perfect now ! 👏🏻

**Comment by unRARed** [comment]
Same problem in Rails 5 here with `config.api_only = true`. Also tried standardizing everything to `/swagger` with no change in results. Here's my current config:

* Gemfile: `gem 'rswag', '~> 1.1'` also tried `~> 1.0`
* Swagger doc: `./swagger/v1/swagger.json`  
* UI Initializer: `c.swagger_endpoint '/v1/swagger.json', 'API V1 Docs'`
* API Initializer: `c.swagger_root = Rails.root.to_s + '/swagger'`
* API Route: `mount Rswag::Api::Engine => '/swagger'`
* UI Route: `mount Rswag::Ui::Engine => '/swagger'` and no difference with
 `mount Rswag::Ui::Engine, at: '/swagger'`

![Screenshot](http://i.imgur.com/tC6j0Kg.png)

**Comment by pcaplan-teladoc** [comment]
I've also run into this and looks like the problem is inherent to API only application; only solution is to convert to a regular rails app: https://github.com/rails/rails/issues/24264

**Comment by vigneshjayavel** [comment]
This PR fixes the issue https://github.com/domaindrivendev/rswag/pull/107

**Comment by fa-marianovicente** [comment]
It worked! wow, thanks a lot!

**Comment by ChrisSasarman** [comment]
Uncomment this line in _config/application.rb_:
`require "sprockets/railtie"`

**Comment by yzgan** [comment]
@ChrisSasarman You are absolutely right!. It solved! was using older version of rswag (was 1.4.0) in rails API only application and having exactly the same problem.

It can be solved either update to latest version (2.0.5) or Uncomment this line in `config/application.rb`:
```ruby
require "sprockets/railtie"
```
